tests/cli/test_suite.py

```diff
@@ -3436,7 +3440,11 @@ def test_suite_new_profile_runs_notebook_opens_jupyter(
     # },
     # column_value_nonnullity_rule={
     # },
-    # numeric_columns_rule={
+    # numeric_columns_low_precision_rule={
+    #     "false_positive_rate": 0.1,
+    #     "random_seed": 43792,
+    # },
+    # numeric_columns_high_precision_rule={
```

Comment:
@NathanFarmer Please see the comment above regarding adding the rounding examples -- thank you!

Reply:
✅